src/model/chat.test.ts

```diff
@@ -5,6 +5,7 @@ import { ChatModel } from './chat.js';
 const FAKE_RESPONSE: Model.Chat.Response = {
   message: {
     content: 'Hi from fake AI',
+    refusal: null,
```

Comment:
boooo

Reply:
this is required for the openai types to match up w/ response chat messages

Reply:
dexter may live on forever just as a cleanup layer on top of openai's god awful schema design lol